history/epoch-accumulator-test-vectors.md

```diff
@@ -1,6 +1,6 @@
-# Header Gossip Network Test Vectors
+# Epoch Accumulator Test Vectors
 
-This document contains test vectors for the Header Gossip Network to help implementations conform to the specification.
+This document contains test vectors Epoch Accumulator for the Header Network to help implementations conform to the specification.
 
 ## Header Accumulator Encodings
```

Comment:
Can we change all references to `Header Accumulator` to `Epoch Accumulator` throughout this file?  The "Header Accumulator" term is ambiguous since it could easily be understood to refer to the master accumulator as well.

Reply:
I'd suggest to do that in another PR. Because the EIP related to this has even another naming. So lets do the change in one go once we settle on the proper names?